src/Http/Controllers/Character/MarketController.php

```diff
@@ -57,18 +58,37 @@ public function getMarket(int $character_id)
     public function getMarketData(int $character_id)
     {
 
-        $orders = $this->getCharacterMarketOrders($character_id, false);
+        if (! request()->has('all_linked_characters'))
+            return abort(500);
+
+        if (request('all_linked_characters') === 'false')
+            $character_ids = collect($character_id);
+
+        $user_group = User::find($character_id)->group->users
+            ->filter(function ($user) {
+                return $user->name !== 'admin' && $user->id !== 1;
+            })
+            ->pluck('id');
+
+        if (request('all_linked_characters') === 'true')
+            $character_ids = $user_group;
+
+        $orders = $this->getCharacterMarketOrders($character_ids);
 
         return DataTables::of($orders)
             ->addColumn('bs', function ($row) {
 
-                return view('web::partials.marketbuysell', compact('row'))
-                    ->render();
+                if ($row->is_buy_order)
+                    return '<span class="text-red">Buy</span>';
```

Comment:
Can you restore the partial blade ? - so we will keep the controller clean of HTML.

Reply:
yes done: https://github.com/eveseat/web/pull/240/commits/6a011686bda9b58903f6171a18b4713aad84edc6